### PR TITLE
Cleanup WiFi toggle code in tests

### PR DESCRIPTION
**Pull request by jamesonwilliams**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: aws-android-sdk-appsync-tests/src/androidTest/java/com/amazonaws/mobileconnectors/appsync/tests/MultiClientInstrumentationTest.java [hunk 1]
```diff
@@ -502,15 +501,7 @@ public void testMultipleOfflineMutations() {
...
-            new Thread(() -> {
-                // TODO: What on earth is this business?
-                for (int i = 0; i < numberOfLatches; i++) {
-                    Wifi.turnOff();
-                    Sleep.seconds(3);
-                    Wifi.turnOn();
-                    Sleep.seconds(7);
-                }
-            }).start();
+            Wifi.turnOff();
```
Comment (jamesonwilliams) [inline aws-android-sdk-appsync-tests/src/androidTest/java/com/amazonaws/mobileconnectors/appsync/tests/MultiClientInstrumentationTest.java:504]: The JavaDoc above the test says that WiFi is supposed to be off. So? _Just Do It™._

File: aws-android-sdk-appsync-tests/src/androidTest/java/com/amazonaws/mobileconnectors/appsync/tests/MultiClientInstrumentationTest.java [hunk 2]
```diff
@@ -582,12 +573,7 @@ public void testSingleOfflineMutation() {
             String postId = post.id();
             assertNotNull(postId);
 
-            new Thread(() -> {
-                //  TODO: What on earth?
-                Wifi.turnOff();
-                Sleep.seconds(1);
-                Wifi.turnOn();
-            }).start();
+            Wifi.turnOff();
```
Comment (jamesonwilliams) [inline aws-android-sdk-appsync-tests/src/androidTest/java/com/amazonaws/mobileconnectors/appsync/tests/MultiClientInstrumentationTest.java:576]: Same.

File: aws-android-sdk-appsync-tests/src/androidTest/java/com/amazonaws/mobileconnectors/appsync/tests/MultiClientInstrumentationTest.java [hunk 3]
```diff
@@ -655,9 +638,6 @@ public void testClearCache() throws ClearCacheException {
             Posts.list(client, AppSyncResponseFetchers.CACHE_ONLY).get("CACHE");
         assertNotNull(listPostsResponse);
         assertNull(listPostsResponse.data());
-
-        Wifi.turnOn();
-        Sleep.seconds(3);
```
Comment (jamesonwilliams) [inline aws-android-sdk-appsync-tests/src/androidTest/java/com/amazonaws/mobileconnectors/appsync/tests/MultiClientInstrumentationTest.java:660]: This is ensured in `@Before`, we don't need to do it here, too.

File: aws-android-sdk-appsync-tests/src/androidTest/java/com/amazonaws/mobileconnectors/appsync/tests/MultiClientInstrumentationTest.java [hunk 4]
```diff
@@ -722,9 +701,6 @@ public void testClearMutationsCacheOnly() throws ClearCacheException {
             Posts.list(client, AppSyncResponseFetchers.CACHE_ONLY).get("CACHE");
         assertNotNull(cacheResponse);
         assertNotNull(cacheResponse.data());
-
-        Wifi.turnOn();
-        Sleep.seconds(3);
```
Comment (jamesonwilliams) [inline aws-android-sdk-appsync-tests/src/androidTest/java/com/amazonaws/mobileconnectors/appsync/tests/MultiClientInstrumentationTest.java:727]: Ensured in `@Before`, this is duplicative.

File: aws-android-sdk-appsync-tests/src/androidTest/java/com/amazonaws/mobileconnectors/appsync/tests/MultiClientInstrumentationTest.java (hunk 1 above)
Comment (rjuliano) [reply aws-android-sdk-appsync-tests/src/androidTest/java/com/amazonaws/mobileconnectors/appsync/tests/MultiClientInstrumentationTest.java:504]: I guess they must somehow start the emulator without cell service and toggle online state by manipulating the Wifi.

**Comment by jamesonwilliams** [comment]
> Just Ship It

Haha I lol'd

File: aws-android-sdk-appsync-tests/src/androidTest/java/com/amazonaws/mobileconnectors/appsync/tests/MultiClientInstrumentationTest.java (hunk 1 above)
Comment (jamesonwilliams) [reply aws-android-sdk-appsync-tests/src/androidTest/java/com/amazonaws/mobileconnectors/appsync/tests/MultiClientInstrumentationTest.java:504]: I'm not sure about this either! I don't see any reason why the emulator wouldn't have cellular data. Not sure what the design was, here.

**Pull request merged**